test_conformance/vulkan/test_vulkan_interop_image.cpp

```diff
@@ -1360,61 +1365,73 @@ int test_image_common(cl_device_id device_, cl_context context_,
             goto CLEANUP;
         }
         err = clGetDeviceInfo(devices[device_no], CL_DEVICE_UUID_KHR,
-                              CL_UUID_SIZE_KHR, uuid, &extensionSize);
-        if (CL_SUCCESS != err)
+                              CL_UUID_SIZE_KHR, uuid, NULL);
+        test_error_and_cleanup(err, CLEANUP,
+                               "clGetDeviceInfo failed with error");
+
+
+        if (!is_extension_available(devices[device_no],
+                                    "cl_khr_external_memory_opaque_fd"))
         {
-            print_error(err, "clGetDeviceInfo failed with error");
-            goto CLEANUP;
+            log_info(
+                "Device %u does not support cl_khr_external_memory_opaque_fd, "
+                "which is required for this test. Skipping\n",
+                device_no);
+            continue;
```

Comment:
What if cl_khr_external_memory_opaque_fd is not present, but some other handle type extension is present (e.g. cl_khr_external_memory_win32)? Consider either removing the check or adding checks for other handle types too.

Reply:
Done